src/Pages/Home.js

```diff
@@ -74,6 +76,76 @@ function Home(){
                     <button className="Font_style_2">Resume</button>
                 </div>
             </div> 
+            {/* Project section */}
+            <div className="Project_section">
+                <Card
+                className='Project_style_1'
+                hoverable
+                cover={<img alt="example" src= {require('../Assets/spotify.jpg')}  />}>
+                    <Link to="/Project00">
```

Comment:
for the future, this is what’s going to go inside your URL path (i.e. FrankiePortfolio.com/**Project00**), so I recommend you come back to this when you have your projects and update this to a meaningful URL path.